### PR TITLE
DSYS-22/tags-qa

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to task:**
https://labcodes.atlassian.net/browse/DSYS-22 (QA 15/10)

**How to test:**
- With the LabSystem running, go to the "Component/Tag" div in the LabStoryBook, then check if everything is rendered as expected.
- Locally, from labstorybook dir, run `npm test`, then check if all tests are passing.

**Description of your solution:**
This PR fixes the documentation of Tags according to QA from 15/10/20, including "stories" and "playground". The architecture of 'TagPlayground' is refactored into a single dynamic component. Snapshots are udpated.

### CODE REVIEW

File: labsystem/src/Tags/AbstractTag.js [hunk 1]
```diff
@@ -54,8 +57,10 @@ export default class AbstractTag extends React.Component {
           `${color ? ` lab-tag--${color}-${skin}` : ` lab-tag--${skin}`}`
         }
         onClick={this.handleOnClick}
-        onKeyDown={this.handleOnClick}
+        onKeyPress={this.handleOnClick}
```
Comment (lucianoratamero) [inline labsystem/src/Tags/AbstractTag.js:60]: hmm, I hadn't seen this method name here. :thinking: 
maybe change this method name to `handleEvents`? it's weird to use the `handleOnClick` method on the `onKeyPress` event too.
keep the prop names, it's only the component method I'm talking about.

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
besides the method name, I found this bug on the playground. 
playground issue
it could be a bug that's not related only to the 'skin', but to all props.
what I'd recommend would be to reset the state whenever changing components.

File: labsystem/src/Tags/AbstractTag.js (hunk 1 above)
Comment (CavalcanteLucas) [reply labsystem/src/Tags/AbstractTag.js]: ok, fixed

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
The playground bug that I'd reported is still present.

**Comment by CavalcanteLucas** [comment]
Oh! I'm sorry.. I missed one line when refactoring the solution. Thank you for noticing!! Fixed.

**Review by lucianoratamero — APPROVED** [review]

**Pull request merged**